docs/zkapps/advanced-snarkyjs/events.mdx

```diff
@@ -61,6 +61,63 @@ class SmartContract {
 }
 ```
 
+### Fetching Events from an Archive Node
```

Comment:
My suggestions above can be looked at and adapted down here too, if accepted, given this section mirrors the above.

Reply:
@jasongitmail  see the updated PR description about having a single source of the archive node content... as best practice, we do not want to mirror or duplicate content